hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -151,9 +151,27 @@
       reset_connections: {rst_ni: "lc"},
       type: "eflash",
       base_addr: "0x20000000",
-      size: "0x80000" },
+      size: "0x80000",
+      inter_signal_list: [
+        { struct: "flash",    // flash_req_t, flash_rsp_t
+          type: "req_rsp",
+          name: "flash_ctrl", // flash_ctrl_i (req), flash_ctrl_o (rsp)
+          act:  "responder",
```

Comment:
i guess this is specifically defined here (the whole inter_signal_list) because the eflash wrapper itself does not have it's own hjson, is that right?

So if you imagine, in the future, the flash_phy becomes its own module that wraps the memory, this would then be absorbed into the `flash_phy.hjson`, is that right?

Reply:
Right.